src/Util.jl

```diff
@@ -38,65 +39,3 @@ end
     Ar = reinterpret(reshape, Float64, parent(A))
     return reshape(Ar, 3, 3, size(A)...)    # make sure this doesn't mess up indexing
 end
-
```

Comment:
Cleans things up to see these go. Going from \cdot to * for S \cdot B confused me at first, but this all makes sense.

Reply:
With the move to abstract polynomials,  I observed that `S \cdot B ` became valid syntax without defining those `dot` overloads, and it's better to avoid "type piracy" where possible.